src/modules/users/components/user-create.js

```diff
@@ -0,0 +1,18 @@
+import React from "react";
+import { connect } from "react-redux";
+
+import { UserCreateForm } from "modules/users/components/user-form";
+import { UserActionCreators } from "modules/users/user-actions";
+
+const DumbUserCreate = ({ saveUser }) => <UserCreateForm onSubmit={saveUser} />;
```

Comment:
Doesn't matter now but it's more common to use something like `UserCreateComponent`. We actually don't use `Dumb*`. :slightly_smiling_face:

Reply:
:) Will rename that in version 2.0 :package: